react-bricks/bricks/react-bricks-ui/singleColumnContent/Code/PrismCode.tsx

```
@@ -0,0 +1,64 @@
+
+  React.useEffect(() => {
+    highlight()
+  }, [dataLine])
+
+  const dataLineObj = dataLine ? { 'data-line': dataLine } : {}
+
+  return (
+    <>
+      <Styles />
+      <style
+        dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)